packages/utils/src/pick.ts

```diff
@@ -1,9 +1,14 @@
 /**
- * Pick keys from object.
+ * The `pick` function creates a new object by selecting specified keys from
+ * the original object. It takes an object and an array of keys to include,
+ * returning a new object that contains only the properties specified in the
+ * keys array. This is useful for extracting relevant data from an object while
+ * ignoring other properties.
```

Comment:
**thought:** Don't really know if we need to express why the function is useful.